internal/commands/pull/command.go

```diff
@@ -60,23 +62,47 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return err
 	}
 
-	pathTarget, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
+	clientZipPkgs, err := cmd.inputs.resolveClient(ui, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	pathRelative, err := filepath.Rel(profile.WorkingDirectory, pathTarget)
+	pathTarget, zipPkg, err := cmd.doExport(profile, clients.Realm, app.GroupID, app.ID)
 	if err != nil {
 		return err
 	}
 
-	proceed, err := checkAppDestination(ui, pathTarget)
+	if clientZipPkgs != nil {
```

Comment:
i'd prefer _not_ to use the `nil`-presence of a map to decide to something.  could this just be `len(clientZipPkgs) == 0`?  this would technically allow the user to not select anything from the multi-prompt and then subsequently prompt them if they're ok to write contents to a directory, at which point they'll write nothing to that spot.  seems like a clunky UX to me